paf-mvp-core-js/src/express/node.ts

```diff
@@ -37,16 +37,20 @@ export class Node implements INode {
 
   constructor(
     hostName: string,
-    identity: IdentityConfig,
+    protected identity: IdentityConfig,
```

Comment:
Can we move the property definition out of the constructor here, please?

Like I said in other comments, we should agree on a convention regarding that. My point is that mixing properties in both the class declaration and its constructor makes things less simple to read. Even if it is more verbose, it is preferable (IMHO) to have all the declarations in the same place so that we can quickly see the dependencies and understand fasted the dependencies of the class.